pcsx2/gui/MainMenuClicks.cpp

```diff
@@ -158,12 +158,12 @@ wxWindowID SwapOrReset_Iso( wxWindow* owner, IScopedCoreThread& core_control, co
 		}
 	}
 
-	g_Conf->CdvdSource = CDVDsrc_Iso;
+	g_Conf->CdvdSource = CDVD_SourceType::CDVDsrc_Iso;
 	SysUpdateIsoSrcFile( isoFilename );
 	if( result == wxID_RESET )
 	{
 		core_control.DisallowResume();
-		sApp.SysExecute( CDVDsrc_Iso );
+		sApp.SysExecute(CDVD_SourceType::CDVDsrc_Iso );
```

Comment:
Spacing around parenthesis is inconsistent - either there are spaces on each side or there are none. Same applies to some of the other changes.